salt/metalk8s/defaults.yaml

```diff
@@ -27,12 +27,105 @@ kubeadm_preflight:
 
 repo:
   online_mode: true
-  runc:
-    version: latest
-  containerd:
-    version: latest
-  kubernetes:
-    version: 1.11.7
+  local_mode: false
+  base_path: /srv/scality/metalk8s-2.0/packages
+  host: 127.0.0.1
+  port: 8080
+  repositories:
+    metalk8s-base:
+      humanname: CentOS - Base
+      gpgcheck: 1
+      gpgkeys:
+        - RPM-GPG-KEY-metalk8s-base-7_1
+      repo_gpg_check: 0
+      enabled: 1
```

Comment:
Since we use `fromrepo` everywhere we should enable 0 repo so if someone do a command he only have is own repositories.

For me we should either enable all repos and not use `fromrepo` or disable all repo but always use `fromrepo`

Reply:
Agree